### PR TITLE
Build script improvements

### PR DESCRIPTION
**Pull request by chancancode**
This doesn't fix the CI failure on master, but helps shed some light on the issue
- [x] The current verison of phantomjs no longer prints noisy messages to stderr, dropping the check
- [x] Don't supress phantomjs error message when it crashes
- [x] Retry up to 3 times when phantomjs crashes
- [x] Use `minidump_stackwalk` to print the phantomjs stack trace when it crashes

### CODE REVIEW

**Comment by chancancode** [comment]
As seen in https://travis-ci.org/emberjs/ember.js/jobs/105987201

**Comment by chancancode** [comment]
Better diff: https://github.com/emberjs/ember.js/pull/12878/files?w=1

**Comment by chancancode** [comment]
cc @rwjblue

File: .travis.yml [hunk 1]
```diff
@@ -10,11 +10,19 @@ cache:
...
 
 before_install:
   - "npm config set spin false"
   - "npm --version"
   - "phantomjs --version"
+  - "rm -f /tmp/*.dmp"
+  - "mkdir -p phantomjs"
+  - >
+    if [ ! -d ./phantomjs/`phantomjs --version`-linux-x86_64-symbols ]; then
+      wget -O /tmp/phantomjs-`phantomjs --version`-linux-x86_64-symbols.tar.bz2 https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-`phantomjs --version`-linux-x86_64-symbols.tar.bz2 &&
```
Comment (rwjblue) [inline .travis.yml]: What is this doing? What are we downloading here?

**Comment by raytiley** [comment]
Looks like it's downloading the symbols for phantom so it can display relevant details when phantom crashes.

**Comment by rwjblue** [comment]
Thanks @raytiley

**Comment by chancancode** [comment]
Confirm. It's basically what the phantom crash message asked us to do:

```
PhantomJS has crashed. Please read the crash reporting guide at https://github.com/ariya/phantomjs/wiki/Crash-Reporting and file a bug report at https://github.com/ariya/phantomjs/issues/new with the crash dump file attached: /tmp/22c4835e-edf0-12f9-30306b7d-085dc9ef.dmp
```

(It's supposed to be caching it in the travis build cache, but I can't tell if it's working)

**Pull request merged**